### PR TITLE
[FIX] developer: document changes of message_post API

### PR DESCRIPTION
**Pull request by mart-e**

### CODE REVIEW

File: content/developer/reference/backend/mixins.rst [hunk 1]
```diff
@@ -81,8 +81,8 @@ to manage followers on your record:
     Post a new message in an existing thread, returning the new
     mail.message ID.
 
-    :param str body: body of the message, usually raw HTML that will
-        be sanitized
+    :param str,Markup body: body of the message. Will be escaped if `str`. Use
+        a :class:`~markupsafe.Markup` object for HTML content.
```
[the remark this answers is not in the document]
Comment (AntoineVDV) [reply content/developer/reference/backend/mixins.rst:85]: ...which we should avoid.

**Comment by AntoineVDV** [comment]
@robodoo delegate+

**Comment by Feyensv** [comment]
why don't we autodocument those methods instead ? :thinking:

**Comment by mart-e** [comment]
robodoo r+

**Pull request closed without merging**